packages/cli/src/cli/ink-cli/AGENTS.md

```
@@ -0,0 +1,181 @@
+  toolResult?: string
+  toolStatus?: 'running' | 'finished'
+  isError?: boolean              // Tool execution failed
+  styledType?: StyledMessageType // config, stats, help, session-list, etc.
+  styledData?: StyledData
+  isContinuation?: boolean       // Split message continuation
+}
+```
+
+## Component Hierarchy
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)